### DISCUSSION TITLE
A Search history in the Overlay

### DISCUSSION THREAD

**Discussion by Slayer418** [SnosMe/awakened-poe-trade#1027] [Ideas]
Title basically. A simple "Previous search" button would work but a more complete history could be nice too.

**Comment by SnosMe** [comment]
Yeah, I also have this idea, but don't find it useful.
Can you as example show me items/searches where you need to backtrack between them?

![image](https://github.com/SnosMe/awakened-poe-trade/assets/4292308/6703e9f0-ff64-4bff-9a2f-2ff583c89025)

**Reply by Slayer418** [reply]
Well one case where it would be useful to me is when I'm price checking a rare item but gotta do a quick currency price check. 

That or simply just misclicking and voiding a search.
